cmd/kyma/apply/function/function.go

```diff
@@ -122,6 +131,22 @@ func (c *command) Run() error {
 	return mgr.Do(ctx, options)
 }
 
+func (c *command) kymaHostAddress() string {
+	var url string
+	vs, err := c.K8s.Istio().NetworkingV1alpha3().VirtualServices("kyma-system").Get(context.Background(), "apiserver-proxy", v1.GetOptions{})
+	switch {
+	case err != nil:
+		fmt.Printf("Unable to read the Kyma host URL due to error: %s. \n%s\r\n", err.Error(),
```

Comment:
Please do not print errors and then continue execution as if nothing happened. This will then fail further down the line and the printed output will be confusing and make it very hard to debug what is really going on.

Reply:
This function represents additional and optional functionality. Cli may work fine without obtained the kyma address. But you are right that debugging (mostly output log) may be harder to understand. So I will add another Step to log this error as a warning during the working step

Reply:
[Done](https://github.com/kyma-project/cli/pull/838/files#diff-eeb658a08e2c3a583900fa0a93f72515685f64d39eff93dd12c434d50ed04489R99-R102)